iocore/net/test_certlookup.cc

```diff
@@ -206,7 +206,8 @@ SSLReleaseContext(SSL_CTX *ctx)
 int
 main(int argc, const char **argv)
 {
-  diags = new Diags(NULL, NULL, stdout);
+  BaseLogFile *blf = new BaseLogFile("stdout");
+  diags = new Diags(NULL, NULL, blf);
```

Comment:
I see this pattern a lot. Maybe `Diags` should have a constructor overload that takes a string and does the `BaseLogFile` construction.